controllers/openstackcluster_controller.go

```diff
@@ -96,7 +96,7 @@ func (r *OpenStackClusterReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	log = log.WithValues("cluster", cluster.Name)
 
 	if annotations.IsPaused(cluster, openStackCluster) {
-		log.Info("OpenStackCluster or linked Cluster is marked as paused. Won't reconcile")
```

Comment:
I think the present tense is correct in this case: it is describing the current, ongoing state of the object.

You could even leave it alone, I guess, but I *might* change `Won't reconcile` to `Not reconciling`. Or maybe:

Not reconciling because OpenStackCluster or linked Cluster is marked as paused

This is marginal and up to you.

Reply:
I think 'is marked as paused' is important here. Putting myself in the place of somebody reading it, the present tense implies (to me, at least) that we're reporting the current state. The past tense might suggest we're reporting that we just changed it.